### PR TITLE
set connect, socket and query timeouts

### PR DESCRIPTION
**Pull request by malaffoon**
Generally want 30s query timeout ... but there are exceptions.

### CODE REVIEW

File: admin-service/src/main/resources/application.yml [hunk 1]
```diff
@@ -22,31 +22,39 @@ spring:
...
     username: root
     password:
     testWhileIdle: true
     validationQuery: SELECT 1
+    validationQueryTimeout: 10000
     driverClassName: com.mysql.jdbc.Driver
     initialize: false
     maxActive: 10
     initialSize: 4
+    jdbcInterceptors: "QueryTimeoutInterceptor(queryTimeout=30)"
```
Comment (malaffoon) [inline admin-service/src/main/resources/application.yml]: Basic queries get the 30s treatment.

File: admin-service/src/main/resources/application.yml [hunk 2]
```diff
@@ -22,31 +22,39 @@ spring:
...
     username: root
     password:
     testWhileIdle: true
     validationQuery: SELECT 1
+    validationQueryTimeout: 10000
     driverClassName: com.mysql.jdbc.Driver
     initialize: false
     maxActive: 10
     initialSize: 4
+    jdbcInterceptors: "QueryTimeoutInterceptor(queryTimeout=60)"
```
Comment (malaffoon) [inline admin-service/src/main/resources/application.yml]: But we've seen the occasional slow query hitting admin stuff so let's go 60s here.

File: aggregate-service/src/main/resources/application.yml [hunk 3]
```diff
@@ -5,25 +5,6 @@ management:
...
-    # there must be a default region when running locally
-    region:
-      auto: false
-      static: us-west-2
-    # disable CloudFormation stuff
-    stack:
-      auto: false
-    credentials:
-      instance-profile: false
-
```
Comment (malaffoon) [inline aggregate-service/src/main/resources/application.yml]: i alphabetized things

File: aggregate-service/src/main/resources/application.yml [hunk 4]
```diff
@@ -70,26 +85,14 @@ spring:
 
   olap_datasource:
     url: "\
-       jdbc:redshift://${spring.olap_datasource.url-server:localhost:5432}/${spring.olap_datasource.url-db:dev}\
-       ?ApplicationName=rdw-reporting-aggregate-service\
-       "
+      jdbc:redshift://${spring.olap_datasource.url-server:localhost:5432}/${spring.olap_datasource.url-db:dev}\
+      ?ApplicationName=rdw-reporting-aggregate-service\
+      &loginTimeout=${spring.olap_datasource.connect-timeout:10}\
+      &socketTimeout=${spring.olap_datasource.socket-timeout:0}\
```
Comment (malaffoon) [inline aggregate-service/src/main/resources/application.yml]: notice that olap has no timeout enforced; not sure i like this but we agreed to it once upon a time

File: webapp/src/main/resources/application.yml [hunk 5]
```diff
@@ -7,6 +7,33 @@ management:
...
+    name: California
+  user-guide-url: https://portal.smarterbalanced.org/library/en/reporting-system-user-guide.pdf
+  # english-learners should be lep or elas
+  english-learners:
+    - elas
+
+  target-report:
+    insufficient-data-cutoff: 0.2
+    min-number-of-students: 0
+
```
Comment (malaffoon) [inline webapp/src/main/resources/application.yml]: alphabetized again ... perhaps i should take some OCD drugs

File: report-processor/src/main/resources/application.yml [hunk 6]
```diff
@@ -88,20 +88,29 @@ spring:
...
     password:
     testWhileIdle: true
     validationQuery: SELECT 1
+    validationQueryTimeout: 10000
     driverClassName: com.mysql.jdbc.Driver
     initialize: false
+    maxActive: 20
+    initialSize: 4
+    maxWait: 10000
+    jdbcInterceptors: "QueryTimeoutInterceptor(queryTimeout=30)"
```
Comment (agorina) [inline report-processor/src/main/resources/application.yml]: This I think needs to be longer than 30 sec.  There are some  38 seconds in production.

File: report-processor/src/main/resources/application.yml [hunk 7]
```diff
@@ -88,20 +88,29 @@ spring:
...
     password:
     testWhileIdle: true
     validationQuery: SELECT 1
+    validationQueryTimeout: 10000
     driverClassName: com.mysql.jdbc.Driver
     initialize: false
+    maxActive: 20
+    initialSize: 4
+    maxWait: 10000
+    jdbcInterceptors: "QueryTimeoutInterceptor(queryTimeout=120)"
```
Comment (malaffoon) [inline report-processor/src/main/resources/application.yml]: There are some queries (pulling a school's worth of student results for PDF generation) that can be slow. We should fix the queries but, for now, give it some time.

**Pull request merged**